doc/holochain_101/src/planning_a_dapp.md

```diff
@@ -52,21 +52,25 @@ Encryption is a two way process, so the right key enables one to decrypt an encr
 
 ## Data access paradigms
 The following are five data access paradigms. Note that in real-world scenarios it is common to mix these options by combining separate dApps. 
-In instances when many separate dApps are needed to share data, Holochain supports bridging between dApps
+In instances when many separate dApps are needed to share data, Holochain supports bridging between dApps.
 Bridges between two networks with different data privacy models specify who can use the bridge, what data crosses the bridge, and tasks that might run in response to the bridge (e.g. notifications)
 
-Default model for Holochain data is public data shard on a public network, and every Holochain dApp has its own network and data, and creates networks for user-participants as soon as they join a dApp. 
+The default model for Holochain data is public data shared on a public network, and every Holochain dApp has its own network and data, and creates networks for user-participants as soon as they join a dApp. 
 The dApp code sets sharing and verification rules. 
 
 ### Public, shared data on a public network
 
 Public data works like Bittorrent: 
 
-Anybody can join a network
-anybody can request any data they want from the network
-any data is available as long as at least one person is sharing it
-if some data is not shared by enough people, a new random person on the network must share it
-there is no "local only" data  
+   anybody can join a network
```

Comment:
Actually we should changes these to bulleted list
```suggestion
   - anybody can join a network
   - ...
```

Reply:
I agree, a bullet list would look better... Should I insert an additional commit with such a change? I also propose that the items in 'Basic expectations for dApps' section (higher in the document) are also converted to a bullet list.